src/tests/integration_test/chat_test.cljs

```diff
@@ -1,96 +1,79 @@
 (ns tests.integration-test.chat-test
   (:require
-    [cljs.test :refer [deftest is]]
-    [day8.re-frame.test :as rf-test]
+    [cljs.test :refer [deftest is use-fixtures]]
     legacy.status-im.events
-    [legacy.status-im.multiaccounts.logout.core :as logout]
     legacy.status-im.subs.root
+    [promesa.core :as p]
     [re-frame.core :as rf]
     [status-im.constants :as constants]
     status-im.events
     status-im.navigation.core
     status-im.subs.root
     [test-helpers.integration :as h]))
 
+(use-fixtures :each (h/fixture-logged))
+
 (def chat-id
   "0x0402905bed83f0bbf993cee8239012ccb1a8bc86907ead834c1e38476a0eda71414eed0e25f525f270592a2eebb01c9119a4ed6429ba114e51f5cb0a28dae1adfd")
 
 (deftest one-to-one-chat-test
-  (h/log-headline :one-to-one-chat-test)
-  (rf-test/run-test-async
-   (h/with-app-initialized
-    (h/with-account
-     (rf/dispatch-sync [:chat.ui/start-chat chat-id]) ;; start a new chat
-     (rf-test/wait-for
-       [:chat/one-to-one-chat-created]
-       (rf/dispatch-sync [:chat/navigate-to-chat chat-id])
-       (is (= chat-id @(rf/subscribe [:chats/current-chat-id])))
-       (h/logout)
-       (rf-test/wait-for [::logout/logout-method]))))))
+  (h/integration-test ::one-to-one-chat
+    (fn []
+      (p/do
+        (rf/dispatch-sync [:chat.ui/start-chat chat-id])
+        (h/wait-for [:chat/one-to-one-chat-created])
+        (rf/dispatch-sync [:chat/navigate-to-chat chat-id])
+        (is (= chat-id @(rf/subscribe [:chats/current-chat-id])))))))
 
 (deftest delete-chat-test
-  (h/log-headline :delete-chat-test)
-  (rf-test/run-test-async
-   (h/with-app-initialized
-    (h/with-account
-     (rf/dispatch-sync [:chat.ui/start-chat chat-id]) ;; start a new chat
-     (rf-test/wait-for
-       [:chat/one-to-one-chat-created]
-       (rf/dispatch-sync [:chat/navigate-to-chat chat-id])
-       (is (= chat-id @(rf/subscribe [:chats/current-chat-id])))
-       (is @(rf/subscribe [:chats/chat chat-id]))
-       (rf/dispatch-sync [:chat.ui/show-remove-confirmation chat-id])
-       (rf/dispatch-sync [:chat.ui/remove-chat chat-id])
-       (h/logout)
-       (rf-test/wait-for [::logout/logout-method]))))))
+  (h/integration-test ::delete-chat
+    (fn []
+      (p/do
+        (rf/dispatch-sync [:chat.ui/start-chat chat-id])
+        (h/wait-for [:chat/one-to-one-chat-created])
+        (rf/dispatch-sync [:chat/navigate-to-chat chat-id])
+        (is (= chat-id @(rf/subscribe [:chats/current-chat-id])))
+        (is @(rf/subscribe [:chats/chat chat-id]))
+        (rf/dispatch-sync [:chat.ui/show-remove-confirmation chat-id])
+        (rf/dispatch-sync [:chat.ui/remove-chat chat-id])))))
```

Comment:
why do we need to clean-up the chat if we're not asserting anything afterwards?

Reply:
This is beyond my focus in this PR @clauxx. I just kept the tests identical in behavior.